doc/reconstruction_models_list.rst

```diff
@@ -67,137 +67,139 @@
-     - `Tuch 2004 <https://doi.org/10.1002/mrm.20279>`__, `Descoteaux 2007 <https://www.ncbi.nlm.nih.gov/pubmed/17763358>`__, `Tristan-Vega 2010 <https://doi.org/10.1007/978-3-642-04271-3_51>`__
+     - At least 64 directions, maximum b-values 3000-4000 s/mm^2, multi-shell, isotropic voxel size
+     - :cite:t:`Tuch2004`, :cite:t:`Descoteaux2007`, :cite:t:`TristanVega2009b`
    * - :ref:`SHORE <sphx_glr_examples_built_reconstruction_reconst_shore.py>`
      - :bdg-danger:`No`
      - :bdg-success:`Yes`
      - :bdg-danger:`No`
      - Multi-shell HARDI data (500, 1000, and 2000 s/mm^2; minimum 2 non-zero b-shells) or DSI (514 images in a cube of five lattice-units, one b=0)
-     - Merlet 2013, Özarslan 2009, Özarslan 2008
+     - :cite:t`Merlet2013`, :cite:t:`Ozarslan2008`, :cite:t`Ozarslan2009`
    * - :ref:`MAP-MRI <sphx_glr_examples_built_reconstruction_reconst_mapmri.py>`
      - :bdg-danger:`No`
      - :bdg-success:`Yes`
      - :bdg-danger:`No`
-     - Six unit sphere shells with b = 1000, 2000, 3000, 4000, 5000, 6000 s/mm^2 along 19, 32, 56, 87, 125, and 170 directions (see `Olson 2019 <https://doi.org/10.1016/j.neuroimage.2019.05.078>`__ for candidate sub-sampling schemes)
-     - `Ozarslan 2013 <https://doi.org/10.1016%2Fj.neuroimage.2013.04.016>`__, `Olson 2019 <https://doi.org/10.1016/j.neuroimage.2019.05.078>`__
+     - Six unit sphere shells with b = 1000, 2000, 3000, 4000, 5000, 6000 s/mm^2 along 19, 32, 56, 87, 125, and 170 directions (see :cite:t:`Olson2019` for candidate sub-sampling schemes)
+     - :cite:t:`Ozarslan2013`, :cite:t:`Olson2019`
    * - :ref:`MAP+ Constraints <sphx_glr_examples_built_reconstruction_reconst_mapmri.py>`
      - :bdg-danger:`No`
      - :bdg-success:`Yes`
      - :bdg-danger:`No`
      -
-     - `Tom Dela Haije <https://doi.org/10.1016/j.neuroimage.2019.116405>`__
+     - :cite:t:`DelaHaije2020`
    * - MAPL
      - :bdg-danger:`No`
      - :bdg-success:`Yes`
      - :bdg-danger:`No`
-     - Multi-shell similar to WU-Minn HCP, with minimum of 60 samples from 2 shells b-value 1000 and 3000s/mm^2
-     - `Fick 2016 <https://doi.org/10.1016/j.neuroimage.2016.03.046>`__
+     - Multi-shell similar to WU-Minn HCP, with minimum of 60 samples from 2 shells b-value 1000 and 3000 s/mm^2
+     - :cite:t:`Fick2016b`
    * - :ref:`CSD <sphx_glr_examples_built_reconstruction_reconst_csd.py>`
      - :bdg-success:`Yes`
      - :bdg-danger:`No`
      - :bdg-danger:`No`
-     - Minimum: 20 gradient directions and a b-value of 1000 s/mm^2; benefits additionally from 60 direction HARDI data with b-value = 3000s/mm^2 or multi-shell
-     - Tournier 2017, Descoteaux 2008, Tournier 2007
+     - Minimum: 20 gradient directions and a b-value of 1000 s/mm^2; benefits additionally from 60 direction HARDI data with b-value = 3000 s/mm^2 or multi-shell
+     - :cite:t:`Tournier2004`, :cite:t:`Tournier2007`, :cite:t:Descoteaux2007`
```

Comment:
Did not find Tournier 2017 (https://github.com/dipy/dipy/pull/3396/files#diff-abd4df353a7296be6198ba14438aa9418e584ecee676a48841efad39542c6087L142); so changed to what I deemed the appropriate citation for CSD (Tournier 2004).

Reply:
Thanks